cpp/ql/src/semmle/code/cpp/exprs/Expr.qll

```diff
@@ -1043,6 +1107,10 @@ class NoExceptExpr extends Expr, @noexceptexpr {
 /**
  * A C++17 fold expression. This will only appear in an uninstantiated template; any instantiations
  * of the template will instead contain the sequence of expressions given by expanding the fold.
+ * ```
+ * template < typename... T >
+ * auto sum ( T � t ) { return ( t + ... + 0 ); }
```

Comment:
The symbol between `T` and `t` isn't rendering sensibly for me.

Reply:
A Unicode ellipsis, perhaps?  Anyway, fixed it.